i18n/en-US.properties

```diff
@@ -137,7 +137,7 @@ be.contentSidebar.activityFeed.taskNew.taskAssignmentCompleted = Completed
 # Error message when we failed to load the collaborators when user tries to edit a task
 be.contentSidebar.activityFeed.taskNew.taskCollaboratorLoadErrorMessage = An error has occurred while loading collaborators for this task. Please try again.
 # Label and date for task due date
-be.contentSidebar.activityFeed.taskNew.taskDueDateLabel = Due: {date}
+be.contentSidebar.activityFeed.taskNew.taskDueDateLabel = DUE: {date}
```

Comment:
let's not change this one, and instead use the message below `be.contentSidebar.activityFeed.taskNew.taskFeedStatusDue`